rindexer_rust_playground/src/rindexer_lib/indexers/rindexer_playground/playground_types_filter.rs

```diff
@@ -1,176 +1,94 @@
 #![allow(non_snake_case)]
-use std::{path::PathBuf, sync::Arc};
```

Comment:
files got messed up maybe just need to add formatter? we should generate the file for people nicely

Reply:
Fixed